torch/jit/__init__.py

```diff
@@ -1274,7 +1274,7 @@ class _ConstModuleList(ScriptModule):
     def __init__(self, modules):
         super(_ConstModuleList, self).__init__()
         for i, module in enumerate(modules):
-            self.add_module(str(i), module)
+            setattr(self, str(i), module)
```

Comment:
this seems like it has a chance to regress later if the parent class changes it's `add_module`. why is this required?

Reply:
It was being used to convert the `module` to a weak script module if possible, the latest update lifts the check out to here